hashintel/hash
# Prevent tooltips popping-up while user is editing the cell

## 🌟 What is the purpose of this PR?

This PR prevents tooltips on grid cells from popping up while user is editing the cell to improve the UX.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203301853897838/1203244146505186/f) _(internal)_

## 🔍 What does this change?

- Instead of rendering `GlideGridOverlayPortal` manually, `GlideGrid` components adds it to `body` if it's not there yet.
- Fix a small bg color issue
 
## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open entity editor
1.  Check if the behavior works as expected

## 📹 Demo



https://user-images.githubusercontent.com/39553853/203056092-ddad95f2-ddcf-4a1e-b4e7-ec5574f3fa60.mov
